rustls/src/client/mod.rs

```diff
@@ -603,3 +611,15 @@ pub trait ClientQuicExt {
 
 #[cfg(feature = "quic")]
 impl ClientQuicExt for ClientConnection {}
+
+#[test]
+fn too_small_mtu() {
+    use crate::{ConfigBuilder, RootCertStore};
+    let mut client_config = ConfigBuilder::with_safe_defaults()
+        .for_client()
+        .unwrap()
+        .with_root_certificates(RootCertStore::empty(), &[])
+        .with_no_client_auth();
+    client_config.set_mtu(&Some(1));
```

Comment:
If I had to pick a single value to use, I would pick the maximum disallowed value. I'd also like to see a test with the minimum acceptable value.